src/Notifier.js

```diff
@@ -192,12 +216,22 @@ var Notifier = {
         return enabled === 'true';
     },
 
-    setToolbarHidden: function(hidden) {
+     setToolbarHidden: function(hidden, persistent) {
```

Comment:
It's called when user attempts to close the toolbar. This setting should be persistent. Currently the toolbar keep showing up every reload.
`setToolbarPersistantHidden()` set the information in localStorage